### ISSUE TITLE
Hi engineer

### ISSUE THREAD

**Issue by HamidLaktarash** [SuRGeoNix/Flyleaf#136]
your player is great. but palyer cant play sound of some .ts file
video played but audio doesn't. audio format is mpeg audio

**Comment by SuRGeoNix** [comment]
Hi @HamidLaktarash and thanks! Do you know if ffmpeg (try with ffplay or vlc and av demuxer) can play this file by itself? Can you email me the .ts file for testing?

**Comment by HamidLaktarash** [comment]
yes this video played correctly with ffplay.exe and vlc but not in flyleaf player.please test uploaded video on your way with flyleaf playerI read backend code of flyleaf player and change some parameter and code but it doesn't help me and i haven't any idea
Thanks for your attention.

**Comment by SuRGeoNix** [comment]
@HamidLaktarash Where did you upload the video?

**Comment by HamidLaktarash** [comment]
I'm sorry. my first upload failed because its size larger than max size
I upload again

**Comment by HamidLaktarash** [comment]
keep in mind 
when i run blew ffmpeg command file fix
`./ffmpeg  -i "in.ts" -c copy  -y "D:\out.mp4"`
but it take a long time and a lots of video like this.

**Comment by HamidLaktarash** [comment]
which version of flyleaf you used?

**Comment by SuRGeoNix** [comment]
I tried the latest... I noticed an issue with 3.4.4

**Comment by HamidLaktarash** [comment]
I try  the 3.4.10 version Now 
and fix issue. you solve my problem
thank you very much

**Comment by SuRGeoNix** [comment]
Yes probably that was an old issue with mp3/mp2 codecs. please check the latest next time before submitting an issue.

**Comment by HamidLaktarash** [comment]
Hi, I want to create mosaic app with maximum 16 panel using IPTV live
streaming and it should be stable 24/7. I have Nvidia GPU. is flyleaf
player suitable for this use case?
i can costume it and only use video panel and remove ui from it.
Thanks in advance.

**Comment by HamidLaktarash** [comment]
I forgot. recently I use libvlcsharp for my app. it's ok but some times,
some panel crash and goes freeze and i can not recreate them properly.
